test/SparqlAntlrParserTest.cpp

```diff
@@ -1872,56 +1920,118 @@ TEST(SparqlParser, UpdateQuery) {
   auto Literal = [](std::string s) {
     return TripleComponent::Literal::fromStringRepresentation(std::move(s));
   };
+  auto NoGraph = std::monostate{};
```

Comment:
maybe sonar wants (on my instruction) `noGraph` (lowercase), but maybe it ignores the test files completely.